doc/nrf/libraries/caf/ble_adv.rst

```diff
@@ -122,6 +122,18 @@ The |ble_adv| is used only by Bluetooth Peripheral devices.
 
 The |ble_adv| uses Zephyr's :ref:`zephyr:settings_api` to store the information if the peer for the given local identity uses the Resolvable Private Address (RPA).
 
+Undirected advertising data update
+==================================
+
+The module does not instantly update advertising and scan response payloads when advertising data or scan response data provided by :ref:`bt_le_adv_prov_readme` is modified.
+The module automatically gets new advertising data and scan response data from Bluetooth LE advertising data proviers subsystem only in the following cases:
+
+* Bluetooth LE undirected advertising is started or restarted.
+* Undirected advertising enters grace period.
```

Comment:
```suggestion
* Undirected advertising enters the grace period.
```
Plus link to the grace period section, no?

Reply:
Ok, I will add it (then we need to merge the https://github.com/nrfconnect/sdk-nrf/pull/8457 first